docs/content/extensions.md

```diff
@@ -244,7 +246,11 @@ func (p *PrintlnLogger) Log(ctx context.Context, event logs.EventV1) error {
 	if p.config.Stderr {
 		w = os.Stderr
 	}
-	_, err := fmt.Fprintln(w, event)
+	bs, err := json.Marshal(event)
+	if err != nil {
+		p.manager.UpdatePluginStatus(PluginName, &plugins.Status{State: plugins.StateErr})
```

Comment:
I think we should `return nil` after this -- otherwise, we'll keep running here and print the empty string in line 253.

Reply:
that makes sense I'll add that.